### ISSUE TITLE
request to add chapmanganato

### ISSUE THREAD

**Issue by dgbrewer1989** [manga-py/manga-py#464]
# Request to add a site

I'm requesting that chapmanganato be added to manga-py. It appears that https://manganato.com/ is splitting their hosting between manganato and chapmanganato so when visiting the site it could go to either url. Some of the manga's I download work on manganato but the others have been migrated over to chapmanganato. I suspect that the structure is similar with how manganato is setup but I'm not sure.

Description | Data
---|---
Url to add | https://chapmanganato.com

**Comment by 1271** [comment]
note: similar manganelo.com

**Comment by w108x** [comment]
edit: umm, I just realized that their sites are not that good, so maybe people should user other sites instead, not a site I will be using anymore...

`"WARNING:/manga_py/parser.py:40:2022-12-08 02:17:48,683:Manga url changed! New url: https://chapmanganato.com/manga-xxx"`

^ that is for the domain readmanganato.com, so workaround for now is to replace manga's with the URL chapmanganato to readmanganato and it works for now.

**Comment by dgbrewer1989** [comment]
Yeah I'm questioning if I should use that site anymore for manga. I tried the workaround but it didn't seem to work on my end. I'll try and find another site and just start from there. Any suggestions?

**Comment by 1271** [comment]
I'll deal with this issue tomorrow. Sorry it's taking so long

**Comment by xgorn** [comment]
note: you have to fill a referer under the headers to load the images because its protected.

**Comment by ShinoIro** [comment]
> edit: umm, I just realized that their sites are not that good, so maybe people should user other sites instead, not a site I will be using anymore...
> 
> `"WARNING:/manga_py/parser.py:40:2022-12-08 02:17:48,683:Manga url changed! New url: https://chapmanganato.com/manga-xxx"`
> 
> ^ that is for the domain readmanganato.com, so workaround for now is to replace manga's with the URL chapmanganato to readmanganato and it works for now.

Hi, could you please tell me the step by step on how to do that? I'm having the same problem as well. Thanks in advance

**Comment by w108x** [comment]
> > edit: umm, I just realized that their sites are not that good, so maybe people should user other sites instead, not a site I will be using anymore...
> > `"WARNING:/manga_py/parser.py:40:2022-12-08 02:17:48,683:Manga url changed! New url: https://chapmanganato.com/manga-xxx"`
> > ^ that is for the domain readmanganato.com, so workaround for now is to replace manga's with the URL chapmanganato to readmanganato and it works for now.
> 
> Hi, could you please tell me the step by step on how to do that? I'm having the same problem as well. Thanks in advance

I'm using a different tool called gallery-dl now
